.github/workflows/git.yml

```diff
@@ -0,0 +1,12 @@
+name: Git Checks
+
+on: [pull_request]
+
+jobs:
+  block-fixup:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2.0.0
+      - name: Block Fixup Commit Merge
+        uses: 13rac1/block-fixup-merge-action@v1.1.1
```

Comment:
This is him: https://github.com/marketplace/actions/block-fixup-commit-merge

Reply:
And source: https://github.com/13rac1/block-fixup-merge-action/blob/master/entrypoint.sh